### DISCUSSION TITLE
Using Python to parse files

### DISCUSSION THREAD

**Discussion by lkoll** [CMSgov/price-transparency-guide#605] [Q&A]
Has anyone had any success using Python to parse these larger files? I've been looking into streaming options like json-stream or ijon, but haven't quite been able to get the larger ones yet. Just curious if any particular packages or techniques have been helpful.

**Comment by kulnor** [comment]
Alec at DoltHub has been making goo progress on this. 
Please check https://github.com/dolthub/data-analysis/tree/main/transparency-in-coverage
